index.ts

```diff
@@ -1 +1,5 @@
 export * from './src/BrazeMessageComponent';
+
+// Temporary to fix css prop error in Epic story
```

Comment:
Oh, I'm surprised this is still needed. Isn't the new epic component now imported by `BrazeMessageComponent`?

Reply:
This is for the old external epic. Previously it wasn't imported anywhere so had the css prop error. The css prop error only shows up in VSCode and doesn't affect the Github / TeamCity build so we could remove this.

Reply:
Ah I see, thanks for explaining 👍  Strangely I tried removing this locally and don't see the error (it was in the `ExternalEpic` story right and `yarn tsc` also seems happy. I guess I'm slightly nervous here about `import`ing a story file in one of our main entry points, does this end up adding the story to the prod/npm bundle? Separately I definitely think we should remove the `console.log` just below.